client/my-sites/plans-v2/plans-filter-bar/index.tsx

```diff
@@ -38,7 +40,9 @@ const PlansFilterBar = ( {
 	onProductTypeChange,
 }: Props ) => {
 	const barRef = useRef< HTMLDivElement | null >( null );
-	const hasCrossed = useDetectWindowBoundary( barRef );
+	const isMasterbarVisible = useSelector( masterbarIsVisible );
+	const masterbarOffset = isMasterbarVisible ? 47 : 0;
```

Comment:
Not a blocker, but I'm wondering if the master bar height could be read from the DOM rather than hardcoded.

Reply:
Good idea!